### ISSUE TITLE
Odevzdání úkolu 2

### ISSUE THREAD

**Issue by Dituse** [Dituse/Ukoly#2]
Ahoj přidávám úkol 2, mám tam jeden problém, z endpointu dostávám pouze 5 faktů a ne 10.
@janhn

**Comment by janhn** [comment]
Ahoj Dito,
doporucuji znovu kouknout na dokumentaci toho endpointu (parametr `amount` je podporovan u endpointu `/facts/random`).

**Comment by Dituse** [comment]
Ahoj, děkuju, už jsem to upravila, generuje to 10 faktů, ale občas to jsou jen náhodné znaky, nebo je to v ruštině... @janhn

**Comment by janhn** [comment]
Ano, zda se, ze do databaze pod tim API nekdo pridal nesmysly. Tim se ale nemusime trapit.

**Comment by Dituse** [comment]
Dobrá děkuju, myslela jsem si to, že je to otevřený :)
